homeassistant/components/elmax/strings.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title": "Elmax Cloud Setup",
+  "config": {
+    "step": {
+      "user": {
+        "title": "Account Login",
+        "description": "Please login to the Elmax cloud using your credentials",
+        "data": {
+          "password": "[%key:common::config_flow::data::password%]",
+          "username": "[%key:common::config_flow::data::username%]"
+        }
+      },
+      "panels": {
+        "title": "Panel selection",
+        "description": "Select which panel you would like to control with this integration. Please note that the panel must be ON in order to be configured.",
+        "data": {
+          "panel_name": "Panel Name",
+          "panel_id": "Panel ID",
+          "panel_pin": "PIN Code"
+        }
+      }
+    },
+    "error": {
+      "no_panel_online": "No online Elmax control panel was found.",
+      "bad_auth": "Invalid authentication",
+      "network_error": "A network error occurred",
+      "invalid_pin": "The provided pin is invalid",
+      "unknown_error": "An unexpected error occurred"
```

Comment:
We have a common translation string for unknown error similar to how we use a common string below for already configured.

https://github.com/home-assistant/core/blob/9a46e802b7ab354778fe1cd45f86b623af7507e9/homeassistant/strings.json#L59

Reply:
Nice one! Updated in #61273.